lib/presentation/Form/bloc/myform_state.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:equatable/equatable.dart';
+
+class MyFormState extends Equatable {
+  const MyFormState({required this.isEditing});
+  final bool isEditing;
+
+  @override
+  List<Object?> get props => <Object?>[isEditing];
+
+  MyFormState copyWith({
+    bool? isEditing,
+  }) {
+    return MyFormState(
+      isEditing: isEditing ?? this.isEditing,
+    );
+  }
+}
+
+class MyFormSuccessState extends MyFormState {
+  const MyFormSuccessState() : super(isEditing: true);
+
+  @override
+  List<Object?> get props => <Object?>[isEditing];
+}
+
+class MyFormErrorState extends MyFormState {
```

Comment:
error state it is but doesn't have any variable to store errors. map or somethings